### DISCUSSION TITLE
9.1.2023

### DISCUSSION THREAD

**Discussion by charlieforward9** [charlieforward9/ATHLEAT#1] [Team Meetings]
Attendees: @rensaketh @PrestonGoren @ticogarcia 

Open are to drop thoughts/ideas

**Comment by charlieforward9** [comment]
Data Sources:

## Online 

* Geolocation
* Accelerometers
*  Health 
* Screen time? ( @ticogarcia will look into this )
* ...


## Offline

* Journal
* ...


Finalized scope will be decided by next meeting

**Comment by charlieforward9** [comment]
## Programming Strengths

* Preston: Backend
* Saketh: Backend
* Tico: Frontend
* Charles: Fronend

**Comment by charlieforward9** [comment]
## Weekly Collaborations 

Scheduled for **Wednesdays at 5pm in Marston Basement**

**Comment by ticogarcia** [comment]
Interesting Apple Health data we can keep track of (without any extra devices):
Headphone audio levels
Double support time (walking)
Walking speed
Walking step length
Walking asymetry
Active energy
Resting energy
Flights climbed
Without Apple Health:
Sleep with Sleepcycle app (won't be as accurate compared to using a watch)
My fitnesspal for nutrition tracking


Screen time
https://felixkohlhas.com/projects/screentime/
Details a way of accessing it through the apple database. Doesn't seem too bad but seems to require a Mac.

Things we can keep track of without our phones:
Heart rate
Weight
